sys-apps/sydbox/sydbox-1.0.8.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="ptrace-based sandbox"
+HOMEPAGE="https://git.exherbo.org/sydbox-1.git"
+SRC_URI="https://git.exherbo.org/sydbox-1.git/snapshot/${PN}-1-${PV}.tar.xz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="debug seccomp test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="sys-libs/pinktrace:=
+	debug? ( sys-libs/libunwind:= )"
+DEPEND="${RDEPEND}
+	test? ( !<sys-apps/sandbox-2.13 )"
+
```

Comment:
You may want to sync this up with the exheres definition at https://gitlab.exherbo.org/exherbo/arbor/-/blob/master/packages/sys-apps/sydbox/sydbox-1.0.8-r1.exheres-0 and apply the provided patch as well.